Asterix/Example_param_file.ini

```diff
@@ -55,7 +55,7 @@ onbench=True
     # if True, all the pupils will be created 10x larger than diam_pup_in_pix and 
     # then rebinned to diam_pup_in_pix at the exception of pupils that are read in 
     # a fit file, which are always rebinned from the sixe of the .fits
-    grey_pupils = False
+    grey_pupils = True
```

Comment:
I'd prefer to keep the default at False until we have a discussion with Raphael about it

Reply:
Yes, I opened this PR as a follow-up to exactly this discussion with @rgalicher and Pierre at the meeting yesterday (I should have mentioned this, sorry). I will find him offline to see what he thinks about this.
Overall both of them seemed ok with it as long as we can confirm it (a) improves the simulations and (b) doesn't make hardware ops worse. This is also why I suggested to maybe change the FQPM to the zoom-in sampling prop (which needs the grey pupil) only after creating a matrix with this that runs fine on hardware.

So if we choose to not go ahead with the change in the grey pupil default, I would also prefer not to change the default propagation for the FQPM, since that would return really bad results; and the FQPM remains the reference coronagraph for THD2.

Reply:
@johanmazoyer @ivalaginja Ok to upgrade the FQPM to grey pupil + zoom-in sampling prop once it has been demonstrated that the estimation+correction loop works fine (i.e. with same performance as the current version of FQPM). 
Estimation = all estimator (perfect, PW, is there another?)
correction = all correctors (EFC, etc)

Reply:
ok